WikiClientLibrary/Files/FileRevision.cs

```diff
@@ -27,6 +27,9 @@ public class FileRevision
         /// </summary>
         public WikiPageStub Page { get; internal set; }
 
+        [JsonProperty("extmetadata")]
+        public Dictionary<string, MetadataValue> ExtMetadata { get; set; }
```

Comment:
No need to specify property name in `JsonPropertyAttribute` explicitly. We are already [using lowercase by default](https://github.com/CXuesong/WikiClientLibrary/blob/097a9b57d7b48827f507055b236a33f89cb9817f/WikiClientLibrary/Infrastructures/Json.cs#L167) for all MW API JSON serialization.

Reply:
Please replace this field with some immutable interface (e.g. `IReadOnlyDictionary`), and use `private` setter, since `FileRevision` looks immutable from outside.

Reply:
done!